examples/example.py

```diff
@@ -0,0 +1,23 @@
+from caikit_nlp_client import GrpcClient, GrpcConfig, HttpClient, HttpConfig
+
+text = "What is the boilign point of Nitrogen?"
+
+# Using the http client
+client = HttpClient(HttpConfig(host="localhost", port=8080, tls=False))
```

Comment:
Still unsure whether it's worth it to have `HttpConfig`, `GrpcConfig` classes to just pass a few values to the init functions

Reply:
The config classes are a wee big javaesque. They can be removed, but the side effect is that there will be a lot of values that are optional. I suppose the idimatic python is to use `kwargs``.

Reply:
Let's get rid of these classes, and just use (optional) kwargs. We can do this in another PR